pkgs/by-name/pa/pam_fprint_grosshack/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  pkg-config,
+  meson,
+  ninja,
+  glib,
+  libfprint,
+  polkit,
+  systemd,
+  pam,
+  libpam-wrapper,
+  dbus,
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "pam_fprint_grosshack";
+  version = "0.3.0";
+  outputs = [ "out" ];
+
+  src = fetchFromGitLab {
+    owner = "mishakmak";
+    repo = "pam-fprint-grosshack";
+    tag = "v${version}";
```

finalAttrs is preferred in this case 
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "pam_fprint_grosshack";
  version = "0.3.0";
  outputs = [ "out" ];

  src = fetchFromGitLab {
    owner = "mishakmak";
    repo = "pam-fprint-grosshack";
    tag = "v${finalAttrs.version}";
```